polynote-spark-runtime/src/main/scala/polynote/runtime/spark/reprs/SparkReprsOf.scala

```diff
@@ -277,11 +277,20 @@ object SparkReprsOf extends LowPrioritySparkReprsOf {
         val prototype = arr.head
         val rowEncoder = RowEncoder(prototype.schema) // to go from Row to InternalRow
         val (structType, encode) = structDataTypeAndEncoder(prototype.schema) // reuse code from InternalRow
+        val toBytes = rowToBytes(structType, encode)
```

Comment:
Is this change due to Spark 3+?

Reply:
Yes, Spark 3 `ExpressionEncoder` doesn't have the `toRow` method anymore.